api/migrations/versions/cebfdfef31cd_assign_action_editor_email_template.py

```diff
@@ -0,0 +1,73 @@
+"""Adding email templates for assigning an action editor to a journal application
+
+Revision ID: cebfdfef31cd
+Revises: 6a073dd1e30d
+Create Date: 2023-03-02 14:03:29.661901
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = 'cebfdfef31cd'
+down_revision = '6a073dd1e30d'
+
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy import orm
+from sqlalchemy.ext.declarative import declarative_base
+from app import db
+
+Base = declarative_base()
+
+
+
+class EmailTemplate(Base):
+
+    __tablename__ = 'email_template'
+    __table_args__ = {'extend_existing': True}
+
+    id = db.Column(db.Integer(), primary_key=True)
+    key = db.Column(db.String(50), nullable=False)
+    event_id = db.Column(db.Integer(), nullable=True)
+    language = db.Column(db.String(2), nullable=False)
+    template = db.Column(db.String(), nullable=False)
+    subject = db.Column(db.String(), nullable=False)
+
+    def __init__(self, key, event_id, subject, template, language):
+        self.key = key
+        self.event_id = event_id
+        self.subject = subject
+        self.template = template
+        self.language = language
+
+
+def upgrade():
+    Base.metadata.bind = op.get_bind()
+    session = orm.Session(bind=Base.metadata.bind)
+
+    op.execute("""SELECT setval('email_template_id_seq', (SELECT max(id) FROM email_template));""")
```

Comment:
Can you explain what this is doing?
If I had to guess you're trying to ensure the next primary key follows in sequence from the current highest. If I'm right I would say take this out (and from the downgrade) since the db should handle this on it's own (and also having gaps is fine).